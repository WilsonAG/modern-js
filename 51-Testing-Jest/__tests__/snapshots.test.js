const cliente = {
  nombre: 'Wilson Aguilar',
  balance: 500,
  tipo: 'premium'
}

describe('Testing al cliente', () => {
  test('Es Wilson Aguilar', () => {
    expect(cliente).toMatchSnapshot()
  })
  
})
