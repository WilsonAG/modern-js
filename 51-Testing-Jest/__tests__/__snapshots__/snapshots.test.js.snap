// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Wilson Aguilar 1`] = `
Object {
  "balance": 500,
  "nombre": "Wilson Aguilar",
  "tipo": "premium",
}
`;
